### PR TITLE
docs: update hive instructions, ammend test target

### PR DESCRIPTION
**Pull request by fkrause98**
**Motivation**

- Describe how to run hive tests.
**Description**
- Fix readme instructions on how to run hive tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,8 +94,18 @@ and expects a certain response. You can read more about it [here](https://github
 This is an example of a Hive simulation called `ethereum/rpc-compat`, which can be run with:
```
Comment (fmoletta) [inline README.md:94]: We should update this line too

Comment (fkrause98) [reply README.md:94]: Sure!

File: README.md [hunk 2]
```diff
@@ -94,8 +94,18 @@ and expects a certain response. You can read more about it [here](https://github
...
-make run-hive-debug ethereum/rpc-compat
+make run-hive-debug SIMULATION=<simulation> TEST_PATTERN=<test-regex>
 ```
+For example, to run chain id and transaction by hash rpc tests:
+```bash
+make run-hive SIMULATION=ethereum/rpc-compat TEST_PATTERN="/eth_chainId|eth_getTransactionByHash"
+```
+If you want debug output from hive, use the run-hive-debug instead:
+```bash
+make run-hive-debug SIMULATION=ethereum/rpc-compat TEST_PATTERN="*"
```
Comment (fmoletta) [inline README.md:105]: We can omit TEST_PATTERN here

Comment (fkrause98) [reply README.md:105]: I'd leave it for clarity purposes.

File: README.md (hunk 1 above)
Comment (fkrause98) [reply README.md]: Done, changed it a bit, wdy?

File: README.md [hunk 3]
```diff
@@ -91,11 +91,20 @@ make test CRATE=<crate>
 Finally, we have End-to-End tests with hive.
 Hive is a system which simply sends RPC commands to our node,
 and expects a certain response. You can read more about it [here](https://github.com/ethereum/hive/blob/master/docs/overview.md).
-This is an example of a Hive simulation called `ethereum/rpc-compat`, which can be run with:
-
+Hive tests are categorized by "simulations', and can test instances can be filtered with a regex:
```
Comment (fmoletta) [inline README.md:94]:
```suggestion
Hive tests are categorized by "simulations', and test instances can be filtered with a regex:
```

**Review by fmoletta — APPROVED** [review]
🚀

**Pull request merged**